src/index.ts

```diff
@@ -11,45 +11,47 @@ export function activate(context: vscode.ExtensionContext) {
   }
   unibeautify.loadBeautifiers(beautifiers);
   const { supportedLanguages } = unibeautify;
-  return EditProvider.beautifyOptions().then(options => {
-    console.log("Supported languages", supportedLanguages);
-    const enabledLanguages = supportedLanguages.filter(
-      lang => options[lang.name] !== false
-    );
-    console.log("Options", options);
-    console.log("Enabled languages", enabledLanguages);
-    const languageFilters: vscode.DocumentFilter[] = enabledLanguages.reduce(
-      (filters, { vscodeLanguages = [] }) => {
-        const languages: vscode.DocumentFilter[] = vscodeLanguages.map(
-          language => ({ language, scheme: "file" })
-        );
-        return [...filters, ...languages];
-      },
-      [] as vscode.DocumentFilter[]
-    );
-    const patternFilters: vscode.DocumentFilter[] = enabledLanguages
-      .filter(language => language.extensions.length > 0)
-      .map(language => ({
-        pattern: `**/*{${language.extensions.join(",")}}`,
-        scheme: "file",
-      }));
-    const documentSelector: vscode.DocumentSelector = [
-      ...languageFilters,
-      ...patternFilters,
-    ];
-    console.log("Unibeautify documentSelector", documentSelector);
-    const editProvider = new EditProvider();
-    context.subscriptions.push(
-      vscode.languages.registerDocumentFormattingEditProvider(
-        documentSelector,
-        editProvider
-      ),
-      vscode.languages.registerDocumentRangeFormattingEditProvider(
-        documentSelector,
-        editProvider
-      )
-    );
-  });
+  return EditProvider.beautifyOptions().then(
+    (options: LanguageOptionValues) => {
+      console.log("Supported languages", supportedLanguages);
```

Comment:
This is a big change. I think it is only formatting changes?

Reply:
Yep because of the max-line-length it‘s wrapped.